src/appkit.cs

```diff
@@ -2026,7 +2071,7 @@ interface NSButtonCell {
 	}
 	
 	[BaseType (typeof (NSControl))]
-	interface NSButton : NSAccessibilityButton {
+	interface NSButton : NSAccessibilityButton, INSUserInterfaceCompression {
```

Comment:
You want to use `NSUserInterfaceCompression` instead or generator will pick the wrong interface.